logexp/README.md

```diff
@@ -0,0 +1,40 @@
+# LogExp
```

Comment:
I thought of giving a short/crisp name originally. But LogExporter sounds better. Changed it.